### PR TITLE
faq: Add entry for calling op-tee from android .apk apps

### PR DESCRIPTION
**Pull request by vchong**
Signed-off-by: Victor Chong <victor.chong@linaro.org>

### CODE REVIEW

**Comment by jbech-linaro** [comment]
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

**Comment by vchong** [comment]
Tag added. Thanks!

**Comment by vchong** [comment]
Sorry, closed by mistake.

File: faq.md [hunk 1]
```diff
@@ -351,6 +351,20 @@ thread from a TA. If you need to run tasks in parallel, then you should
 probably look into running two TAs or more simultaneously and then let them
 communicate with each other using the TA2TA interface.
 
+### How can I use or call OP-TEE (GlobalPlatform) APIs from an Android (.apk) app?
+Use JNI. First get familiar with [sample_hellojni.html] and make sure you can
+run the sample. After that, replace the C-side Implementation with say the
+[Hello World Trusted Application] or one of the examples in [optee_examples].
```
Comment (igoropaniuk) [inline faq.md]: `linaro-swg/hello_world.git` was fully replaced by `linaro-swg/optee_examples.git` (in both manifest.git and build.git)
I think we should notice only `optee_examples` here  (and btw, it has a copy of original hello_world inside)

Comment (vchong) [reply faq.md]: Removed reference to `linaro-swg/hello_world.git`.

**Comment by jbech-linaro** [comment]
`Reviewed-by: Joakim Bech <joakim.bech@linaro.org>`

**Comment by vchong** [comment]
Tag added. Thanks!

**Pull request merged**